pkg/podexecutor/pebinary.go

```diff
@@ -0,0 +1,305 @@
+// +build windows
+
+package podexecutor
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"k8s.io/client-go/tools/clientcmd"
+
```

Comment:
If you kill these random blank lines, gofmt will sort the imports for easier reading.